doc/user/content/guides/cdc-mysql.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Change Data Capture (MySQL)"
+description: "How to create real-time materialized views from MySQL changelog data with Materialize."
+weight: 10
+menu:
+  main:
+    parent: guides
+---
+
+Change Data Capture (CDC) allows you to track and propagate changes in a MySQL database to downstream consumers based on its binary log (`binlog`). In this guide, we'll cover how to use Materialize to create and efficiently maintain real-time materialized views on top of CDC data.
```

Comment:
I would go with "Change Data Capture (CDC) allows you to track changes in a MySQL database and propagate them to downstream consumers based on the database's binary log" just because of the implicit preposition clash in "track changes in"/"propagate changes from".

Reply:
Ah, in my head the "in" refers to just "changes". I'm okay with changing it, though this would also need to be adjusted in the Postgres CDC guide.